test/fixture/terraform_12/config.yaml

```diff
@@ -10,5 +10,4 @@ suites:
   - git_issue_56__line_ends_with_closing_paren
   - git_issue_66_aws_aliased_provider
   - git_issue_88__corrupt_hcl_file
-  - google_beta_resource_skip
-  - google_vs_google_beta
\ No newline at end of file
+  - google_beta_resource_skip
```

Comment:
removed this test from older providers where setting a specific provider version doesn't work...